Migrate Button component to tailwind (#232)

kakaly opened the pull request:
# Details

Migrated Button component to tailwind

## Description

* Defined new tailwind classes inside tailwind.css

* Defined new colors inside tailwind.js

* Modified Button to use the newly defined css classes

crysfel commented:
Thanks for contributing, I left a couple comments. Good Job!

kakaly commented on src/theme/tailwind.css:
```diff
@@ -1,7 +1,135 @@
 @tailwind components;
 
+/* button */
+.button {
```
Sounds good. Updated the PR accordingly. We can merge it if you think it looks good.

emmawedekind commented:
Hello - what's the status on this!

kakaly commented:
@emmawedekind - I have addressed @crysfel 's comments and updated the changes already. Let me know if you have any comments on this. Otherwise, it is good to merge.

tureey commented:
Everything what was asked has been implemented as I can see. Ready to merge IMHO but wait for more people if so

crysfel commented on tailwind.js:
```diff
@@ -54,21 +53,22 @@ let colors = {
   'primary-lighter': '#E6F5F0',
   'primary-light': '#69D5B1',
-  'primary': '#00bc89',
+  'primary-blur': '#D5FFF4',
```
Please remove `primary-blur` and use `primary-lightest` as is the same value.

kakaly replied:
Done!

crysfel commented:
Build is failing on Travis, can you take a look?

kakaly commented:
@crysfel - fixed it. I modified danger-darker to danger-dark when i resolved merge conflicts by mistake. Reverted it to danger-darker. Should be okay now.
